exercise.js

```diff
@@ -0,0 +1,56 @@
+const addPerson = (request_obj, res) => {
+  if (!request_obj.files) {
+      return res.status(400).send("No files were uploaded.");
+  }
+
+  let Message = '';
+  let first_name = request_obj.body.first_name;
+  let last_name = request_obj.body.last_name;
+  let position = request_obj.body.position;
+  let int_number = request_obj.body.number;
+  let username = request_obj.body.username;
+  let UploadedFile = request_obj.files.image;
+  let image_name = UploadedFile.name;
+  let fileExtension = UploadedFile.mimetype.split('/')[1];
+
```

Comment:
let first_name = request_obj.body.first_name; --- validate for empty // and max length of string

same for 
lastname
 username - check characters - second catch from front-end post
same for UploadedFile